src/mcMd/analyzers/system/Cluster.cpp

```diff
@@ -74,4 +74,68 @@ namespace McMd
       return true;
    }
 
+   Vector Cluster::clusterCOM(int atomTypeInCluster, Boundary* boundaryPtr)
```

Comment:
Change interface to 

Vector Cluster::clusterCOM(int atomTypeInCluster, Boundary const & boundary)

if a const reference is possible, or with a non-const reference Boundary & boundary if necessary.